ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -90,7 +90,7 @@ tests:
       cli: latest
       commands: make test-install-e2e
       dependencies:
-      - env: OO_INDEX
+      - env: OO_BUNDLE
         name: ci-index
```

Comment:
let's rename `ci-bundle` or `ci-index-bundle` or so

Reply:
This is the default name derived from the `bundles:` in line 42
We can change it to numaresources-operator-bundle to make it more explicit

Reply:
yeah, you are right, let's not bother